dpbench/benchmarks/l2_distance/l2_distance_numba_n.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Intel Corp.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+import numba as nb
+
+nb.njit(parallel=False, fastmath=True)
```

Comment:
the njit statement should be added as a decorator to the `l2_distance` function.

Reply:
Good catch. The test did not fail because the missing decorator notation got converted to a nb.njit function call (essentially a no-op). l2_distance function was executed as a python function with a bunch of statements executing on numpy data.